docs/sphinx/source/whatsnew/v0.8.0.rst

```diff
@@ -91,13 +91,23 @@ Enhancements
 * Added *racking_model*, *module_type*, and *temperature_model_parameters* to
   PVSystem, LocalizedPVSystem, SingleAxisTracker, and
   LocalizedSingleAxisTracker repr methods. (:issue:`1027`)
+* Added :py:func:`~pvlib.tracking.calc_axis_tilt` and
+  :py:func:`~pvlib.tracking.calc_cross_axis_tilt` to calculate the
+  tracker axes tilt and the cross-axis tilt angle, which is perpendicular to the
+  tracker axes for trackers on sloped terrain that is not parallel to the axes,
+  such as a north-south tracker on an south-east slope. (:pull:`823`)
```

Comment:
```suggestion
  tracker axis tilt and the cross-axis tilt angle, which is perpendicular to the
  tracker axis azimuth. (:pull:`823`)
```

Reply:
I don't think this is correct for trackers with an axis tilt b/c the cross-axis tilt is measured in the plane perpendicular to the tracker axes (the tx-tz plane where t is the tracker coord-sys). IMO the axis azimuth is a line projected on the horizontal that points in the direction of the tracker axis. So if the tracker-axes are tilted then the cross-axis tilt would not be perpendicular to the axis-azimuth.